### PR TITLE
fix advisory: shell script injection 

### PR DESCRIPTION
**Pull request by Hexilee**

### CODE REVIEW

File: pkg/util/security/escape_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+func TestEscapeBashStr(t *testing.T) {
+	cases := [][]string{
+		{"abc", "abc"},
+		{"test-volume", "test-volume"},
+		{"http://minio.kube-system:9000/minio/dynamic-ce", "http://minio.kube-system:9000/minio/dynamic-ce"},
+		{"$(cat /proc/self/status | grep CapEff > /test.txt)", "$'$(cat /proc/self/status | grep CapEff > /test.txt)'"},
+		{"hel`cat /proc/self/status`lo", "$'hel`cat /proc/self/status`lo'"},
+		{"'h'el`cat /proc/self/status`lo", "$'\\'h\\'el`cat /proc/self/status`lo'"},
+		{"\\'h\\'el`cat /proc/self/status`lo", "$'\\'h\\'el`cat /proc/self/status`lo'"},
```
Comment (davies) [inline pkg/util/security/escape_test.go]: We could implement `unescape` for test.

Comment (Hexilee) [reply pkg/util/security/escape_test.go]: We can straightly `unescape` it by `bash -c "echo %s"`

Comment (davies) [reply pkg/util/security/escape_test.go]: Sounds good

Comment (Hexilee) [reply pkg/util/security/escape_test.go]: Done

**Pull request merged**